src/models/device.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Integer, Float, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from src.db.db import Base
+
+
+class Device(Base):
+    __tablename__ = "devices"
+
+    # columns
+    id = Column(Integer, primary_key=True, index=True)
+    latitude = Column(Float)
+    longitude = Column(Float)
+    device_name = Column(String(100))
+    created_at = Column(DateTime)
+    updated_at = Column(DateTime)
+    user_id = Column(Integer, ForeignKey("users.id"))
+
+    # relationships
+
+    # owner
+    owner = relationship("User", back_populates="devices")
+
+    # owned
+    temperature_data = relationship("Temperature", back_populates="owner")
+    humidity_data = relationship("Humidity", back_populates="owner")
+    motion_data = relationship("Motion", back_populates="owner")
+    button_data = relationship("Button", back_populates="owner")
+    alarm_data = relationship("Alarm", back_populates="owner")
+    notification_data = relationship("Alarm", back_populates="owner")
```

Comment:
Use `backref` instead of `back_populates`.
Like
```python
temperature_data = relationship("Temperature", backref="device")
```

Reply:
It appears backref is legacy - would this be your preference?
https://docs.sqlalchemy.org/en/14/orm/backref.html

Also to confirm, is the functionality ok as it is now and the comments are because backref requires less lines of code?

Thanks!

Reply:
Oh sorry I didn't know that.
I saw that backref and the back_populates will perform the same functionality but backref requires less code.
But it seems it is deprecated, so we should use back_populates.

Reply:
OK